unfetter-discover/unfetter-ui
# Issue 929 (Baselines:  Capability Group Selector)

Fixes unfetter-discover/unfetter#968
Fixes unfetter-discover/unfetter#929

Capability Group selector is the first step of the baselines wizard.  It allows the user to choose from a set of capability groups defined from existing sensor data, but could and should allow for on-the-fly creation of capability groups in this step as well (future issue).  As capability groups are selected, steps are added to the sidebar, and selected capability groups are saved in the wizard ngrx store for use in later steps.  Add and delete buttons are available for each entry only when appropriate.

This PR also has a renaming of the assess3 module to baseline, as well renaming of 'assess', 'assessments', etc to 'baseline' and 'baselines' where appropriate.  Also, code under src/app/models/assess that was baseline-specific has been moved into src/app/models/baseline. 
 There may be some terms not renamed or incorrectly renamed, so please update as you see fit.

## To test:
1. Pull this PR
2. Restart UI:  `docker-compose restart unfetter-ui`
3. Navigate to Baselines
4. Display list of baselines using down triangle near baseline name in sidebar
5. Click Create
6. Enter baseline settings (title, desc, org) and click Continue
7. Select one or more capability groups - they should be added to sidebar as they are selected
8. Once done with capability groups selection, click Continue (empty page will display, expected)
9. Click Categories in the sidebar, and currently selected categories should be pre-selected

NOTE:  'Categories' will be renamed 'Capability Groups' in subsequent PR.
